config/helm/chart/default/values.yaml

```diff
@@ -16,14 +16,6 @@
 platform: "kubernetes"
 
 ### base properties
-
-name: "dynakube"
-apiUrl: ""
-customPullSecret: ""
-skipCertCheck: false
-proxy: ""
-trustedCAs: ""
-networkZone: ""
 enableIstio: false
```

Comment:
is enableIstio and namespaceSelector still used?

Reply:
Dynakube is not created but webhooks and csidriver are. 
Correct me if I'm wrong - they need these fields to work properly:
- webhooks - `namespaceSelector`
- csidriver - `enableIstio`

Reply:
I don't see were these values would be used, I checked the webhook deployment and configurations, as wells as the CSI driver daemonset and configuration

Reply:
Right, I was confused with injection configuration for `applicationMonitoring`, but this should be set up with `dynakube`.